### ISSUE TITLE
Media text should be blank when field is blank

### ISSUE THREAD

**Issue by mwentz** [OrchardCMS/OrchardCore#14382]
When a media field has "Allow media text" enabled, if the Media Text field is empty, the alt text is rendered as the file / path of the image. 

### Expected behavior
The alt tag should be blank when the Media Text field is empty. This is useful for cases where the image is for presentation purposes only and should not be read by a screen reader. Source regarding decorative images: https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by hishamco** [comment]
Could you please send a PR?

**Comment by sebastienros** [comment]
Should we change the UI instead such that the image text is enabled by a checkbox? "Allow" doesn't mean that what is typed is the actual value, but you could provide one. So not setting anything doesn't mean we want an empty value.

Or change the field hint to "Show media text editor" and document this behavior in the UI as a hint.

**Comment by mwentz** [comment]
@sebastienros I don't think that just adding a hint will address the issue since there needs to be a way to include a blank alt tag per accessibility best practices. Everything is working as it should except that when the field is left blank the alt tag should be blank. I don't believe there are any use cases where you would want the filename in the alt tag.

**Comment by sebastienros** [comment]
@mwentz Ok, feel free to create a PR for this, thanks.

**Comment by DrewBrasher** [comment]
If no one is working on this issue I will work on it and create a PR
